lang/lang_en_fr.txt

```diff
@@ -1187,6 +1294,10 @@
 "The printer will start printing a zig-zag line. Rotate the knob until you reach the optimal height. Check the pictures in the handbook (Calibration chapter)."
 "L'imprimante commencera a  imprimer une ligne en zig-zag. Tournez le bouton jusqu'a atteindre la hauteur optimale. Consultez les photos dans le manuel (chapitre Calibration)."
 
+#c=20 r=4
+"Verification failed, remove the filament and try again."
+"Echec de la verification, retirez le filament et reessayez."
```

Comment:
This could be shortened to `Verification a echoue. Enlevez le filament et reessayez.`

Reply:
Thanks again. I will wait a bit before changing it, maybe other will comment as well.

Reply:
"Verification en echec, retirez le filament et reessayez"

Reply:
> Verification en echec, retirez le filament et reessayez

Take this one.